app/controllers/breweries_controller.rb

```diff
@@ -0,0 +1,11 @@
+class BreweriesController < ApplicationController
+  before_action :authenticate_user!
+
```

Comment:
We cleaned out the Breweries Controller because we haven't actually implemented those features yet. We just wanted to establish the migrations/models for breweries in this pull request, so we could make a seed file. We will add more code to this controller later.